--- 1.txt	2023-07-02 22:51:11.294932931 +0000
+++ /dev/fd/63	2023-07-02 23:19:44.697388005 +0000
@@ -1,2 +1 @@
-delete
 stays stays stays stays
--- 2.txt	2023-07-02 22:51:20.959034773 +0000
+++ /dev/fd/63	2023-07-02 23:19:44.705388099 +0000
@@ -1,4 +1,2 @@
 stays stays stays stays
-delete
 stays stays stays stays
-delete
--- 3.txt	2023-07-02 22:51:47.215311601 +0000
+++ /dev/fd/63	2023-07-02 23:19:44.709388146 +0000
@@ -1,4 +1,3 @@
 stays stays stays stays
 stays stays
-delete
 stays stays stays stays
